ecosystem/typescript/sdk/src/aptos_client.ts

```diff
@@ -186,7 +186,7 @@ export class AptosClient {
     query?: { ledgerVersion?: AnyNumber },
   ): Promise<Gen.MoveResource[]> {
     const f = this.client.accounts.getAccountResources.bind({ httpRequest: this.client.request });
-    const out = await paginateWithCursor(f, accountAddress, 1000, query);
+    const out = await paginateWithCursor(f, accountAddress, 9999, query);
```

Comment:
This made me favor the approach of materializing the result at the storage layer wrapper again. lol. Did you finish updating Python and Rust clients?

Reply:
Not yet, I can do that soon though.